guides/Masternode_Upgrade_Cold_Hot_Linux.md

```diff
@@ -0,0 +1,140 @@
+## SHEKEL Cold + Hot wallet MasterNode upgrade guide
+
+> This is a community contributed guide. Feel free to suggest improvements via Issues or opening Pull Requests. Thank you!
+
+
+**!!! This guide is for upgrading an existing SHEKEL COLD & Hot wallet MasterNode using the new Shekel ZeroCoin wallet and chain !!!**
+
+### Requirements
+* Existing version of Shekel-qt on Windows v1.2.0.0 or higher
```

Comment:
1.3.0.0 here?

Reply:
It's saying you need the black 1.2.1.0 wallet and not the old white one as a requirement to upgrade from